File: SalesLibraries.ServiceConnector/Models/Rest/VersionsManagement/ChangesSetRequestData.cs
using System;
using Newtonsoft.Json;
using RestSharp;
using SalesLibraries.ServiceConnector.Models.Rest.Common;

namespace SalesLibraries.ServiceConnector.Models.Rest.VersionsManagement
{
	public class ChangesSetRequestData : IRequestData
	{
		[JsonIgnore]
		public string ModelName => "changes";

		[JsonIgnore]
		public Method Method => Method.PUT;

		public Guid LibraryId { get; set; }
		public string User { get; set; }

		[JsonProperty(TypeNameHandling = TypeNameHandling.None)]
		public ChangeSet[] PendingChanges { get; set; }
	}
}

File: WebSalesLibraries/protected/models/services_data/cadmin/models/versions_management/ChangesSetRequestData.php
<?
	namespace application\models\services_data\cadmin\models\versions_management;

	/**
	 * Class ChangesSetRequestData
	 */
	class ChangesSetRequestData
	{
		public $libraryId;
		public $user;
		/** @var  ChangeSet[] $pendingChanges */
		public $pendingChanges;
	}